### PR TITLE
Use mobiledrawer to show mobile geoeditor version.

### PR DESCRIPTION
**Pull request by simonseyock**

### CODE REVIEW

**Review by jansule — COMMENTED** [review]
Nice work! Looks like some translations might need an update, right?

File: munimap/static/sass/components/_geoeditor.sass [hunk 1]
```diff
@@ -7,18 +7,9 @@
...
-    display: table
-
-  .geoeditor-button-wrap
-    display: table-row
-    width: 100%
-
-  .geoeditor-button
-    display: table-cell
-    width: 100%
+$sidebar-width: 300px // TODO: import value
```
Comment (jansule) [inline munimap/static/sass/components/_geoeditor.sass:10]: should this TODO be done before merging?

**Review by jansule — APPROVED** [review]
Nice work! This now depends on https://github.com/stadt-bielefeld/bielefeldGEOCLIENT/pull/8, right?

File: munimap/static/sass/components/_geoeditor.sass (hunk 1 above)
Comment (simonseyock) [reply munimap/static/sass/components/_geoeditor.sass:10]: good point. I will see if I can do that.

**Review by jansule — APPROVED** [review]

**Pull request merged**